llvm/docs/LangRef.rst

```diff
@@ -20812,10 +20812,10 @@ Arguments:
 The first operand and the result have the same vector of integer type. The
 second operand is the vector mask and has the same number of elements as the
 result vector type. The third operand is the explicit vector length of the
-operation. The fourth argument must be a constant and is a flag to indicate
+operation. The fourth operand must be a constant and is a flag to indicate
 whether the result value of the '``llvm.vp.abs``' intrinsic is a
-:ref:`poison value <poisonvalues>` if the argument is statically or dynamically
-an ``INT_MIN`` value.
+:ref:`poison value <poisonvalues>` if the first argument is statically or
+dynamically an ``INT_MIN`` value.
```

Comment:
This description has a weird mix of operand and argument -- should probably be argument everywhere?

Reply:
LOL - I just recommended that we consistently use argument on another patch as that's what the subsection is called - don't really care which we chose, just want it to be consistent.

Reply:
I was trying to make it consistent with preceding text and still failed.